flask_apscheduler/scheduler.py

```diff
@@ -89,6 +89,10 @@ def start(self, paused=False):
         Start the scheduler.
         :param bool paused: if True, don't start job processing until resume is called.
         """
+
```

Comment:
Can you add this comment to explain why we are doing this validation?

# Flask in debug mode spawns a child process so that it can restart the process each time your code changes,
# the new child process initializes and starts a new APScheduler causing the jobs to run twice.

Reply:
OK，Thx